client/README.md

```diff
@@ -8,8 +8,7 @@ This is the official client layer of Stronghold. It provides a Riker actor model
 - `ListIds`: Returns the `RecordIds` and `RecordHints` from the records contained in the specified Vault
 - `WriteSnapshot`:  Writes a snapshot with a given password, filename and path. If no path is given, the path will default to the `$HOME/.engine/snapshot` folder. The name of the snapshot defaults to `backup.snapshot`.
 - `ReadSnapshot`:   Reads the snapshot with a given password, filename and path. If no path is given, the path will default to the `$HOME/.engine/snapshot` folder. The name of the snapshot defaults to `backup.snapshot`.
-- `WriteData`: Writes data into the record in the specified Vault. If a `RecordId` is not specified, it will write to the head of the vault. This action must be called after `CreateVault` or `InitRecord` must be called first or else this command will replace the data in the record. 
-- `ReadData` - Reads data from the record of the specified Vault. If a `RecordId` is not specified, it will write to the head of the vault.  
+- `WriteData`: Writes data into the record in the specified Vault. If a `RecordId` is not specified, it will write to the head of the vault. This action must be called after `CreateVault` or `InitRecord` must be called first or else this command will replace the data in the record.  
```

Comment:
There is no longer a head to write to.  Write data just writes into the specified location and the record needs to be indicated.